src/query/catalog/src/database.rs

```diff
@@ -174,4 +178,26 @@ pub trait Database: DynClone + Sync + Send {
             self.name()
         )))
     }
+
+    #[async_backtrace::framed]
+    async fn upsert_table_lock_rev(
+        &self,
+        _req: UpsertTableLockRevReq,
+    ) -> Result<UpsertTableLockRevReply> {
+        Err(ErrorCode::Unimplemented(format!(
+            "UnImplement upsert_table_lock_rev in {} Database",
+            self.name()
+        )))
+    }
+
+    #[async_backtrace::framed]
+    async fn delete_table_lock_rev(
+        &self,
+        _req: DeleteTableLockRevReq,
+    ) -> Result<DeleteTableLockRevReply> {
+        Err(ErrorCode::Unimplemented(format!(
+            "UnImplement delete_table_lock_rev in {} Database",
+            self.name()
+        )))
+    }
```

Comment:
Locking/unlocking a table could be just an operation of a `Table`, because it uses a `TableId` to locate a Lock record. It does not need `database` to access a table lock.